Taarafo.Core/Models/PostImpressions/PostImpression.cs

```diff
@@ -0,0 +1,27 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using Taarafo.Core.Models.Posts;
+using Taarafo.Core.Models.Profiles;
+
+namespace Taarafo.Core.Models.PostImpressions
+{
+    public class PostImpression
+    {
+        public Guid Id{ get; set; }
```

Comment:
Let's consider making the `PostId` and `ProfileId` the composite primary key for this value.

Which means, you will need an additional step in your reference setup that looks as follows:

```csharp
 modelBuilder.Entity<PostImpression>()
        .HasKey(c => new { c.PostId, c.ProfileId });
```